### PR TITLE
AGENT-674: Create release periodic jobs for nightly stream

### PR DESCRIPTION
**Pull request by pawanpinjarkar**

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.14.yaml [hunk 1]
```diff
@@ -421,6 +421,27 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn-crun
+- as: e2e-agent-compact-ipv4
+  cluster: build05
+  cron: 0 0 * * 1
```
Comment (andfasano) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.14.yaml:426]: The frequency should be (at least) every 6h/12h initally as suggested in https://docs.ci.openshift.org/docs/architecture/release-gating/#add-a-periodic-job

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 2]
```diff
@@ -970,6 +970,27 @@ tests:
   steps:
     cluster_profile: gcp-openshift-gce-devel-ci-2
     workflow: openshift-e2e-gcp-graceful-shutdown
+- as: e2e-agent-compact-ipv4
+  cluster: build05
```
Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:974]: I would let the `make jobs` decide which build cluster to use.
You can remove this line.

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 3]
```diff
@@ -970,6 +970,27 @@ tests:
...
+  cron: 0 */12 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    workflow: agent-e2e-compact-ipv4
+  timeout: 5h0m0s
+- as: e2e-agent-sno-ipv6
+  cluster: build05
```
Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:981]: Same as above

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 4]
```diff
@@ -970,6 +970,27 @@ tests:
...
+  cron: 0 */12 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    workflow: agent-e2e-sno-ipv6
+  timeout: 5h0m0s
+- as: e2e-agent-ha-dualstack
+  cluster: build05
```
Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:988]: Same as above

**Comment by deepsm007** [comment]
Can we execute rehearsal job to see if your periodics pass as intended?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml (hunk 2 above)
Comment (andfasano) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:974]: build05 is required to use OFCIR

**Pull request merged**